packages/holocron/src/ducks/compose.js

```diff
@@ -52,7 +52,10 @@ export function composeModules(moduleConfigs) {
 
           return Promise.resolve();
         })
-        .catch((error) => error);
+        .catch((error) => {
+          if (error.abortComposeModules) throw error;
+          return error;
+        });
     });
     return Promise.all(modulePromises);
```

Comment:
Wont this promise.all still block until everything is done?

can we somehow abort out of this if the 'right' error is thrown 🤔